promgen/templates/promgen/navbar.html

```diff
@@ -36,7 +36,7 @@
                         <li><a href="{% url 'api:api-root' %}">API</a></li>
                         <li><a href="{% url 'config-targets' %}">Export Targets</a></li>
                         <li><a href="{% url 'api:all-rules' %}">Export Rules</a></li>
-                        <li><a href="{% url 'config-urls' %}">Export URL</a></li>
+                        <li><a href="{% url 'api:all-targets' %}">Export URL</a></li>
```

Comment:
I have had time to test this yet, but just looking at the changes I feel a little bit confused.

Shouldn't this replace `<li><a href="{% url 'config-targets' %}">Export Targets</a></li>` instead?

Reply:
Yes it should. Goof on my part. I'll fix it in a moment 🙇

Reply:
Since there's only the single line in that file, I've rebased and pushed (instead of a separate commit).
I'll likely do a similar change for config-urls in a separate PR and update the tests there as well.